ansible/roles/docker/templates/etc/default/docker.default

```diff
@@ -4,8 +4,11 @@
 #DOCKER="/usr/local/bin/docker"
 
 # Use DOCKER_OPTS to modify the daemon startup options.
-#DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
-DOCKER_OPTS="--storage-driver=devicemapper"
```

Comment:
We can also make the switch to aufs.  How many docker containers do you find yourselves running on each minion, if I may ask?